src/navigation/drawerContent.tsx

```diff
@@ -0,0 +1,151 @@
+
+    const { user, setUserLoggedIn } = useContext(UserContext)
+
+    const logout = async () => {
+        await AsyncStorage.removeItem('token')
+        setUserLoggedIn(false)
+        props.navigation.reset({
+            index: 0,
+            routes: [{ name: routes.LOGIN }]
+        })
+    }
+
+    return (
+        <DrawerContentScrollView {...props}>
+            <Animated.View style={styles.drawerContent}>
+                <Drawer.Section>
+                    <View style={styles.logoSection}>
+                        <Image style={styles.logoIcon} source={Images.logoIcon} />
+                        <Image source={Images.logoText} />
+                    </View>
+                    <Divider style={styles.separator} />
+                    <View style={styles.userInfoSection}>
+                        <Text style={styles.title}>{user ? user.username : 'Loading...'}</Text>
+                        {user && <Text style={styles.subtitle}>User ID: {user.id}</Text>}
+                        {user && <Text style={styles.credits}>{user.credits} credits</Text>}
+                        {user && <View>
+                            <Button
+                                style={styles.button}
+                                contentStyle={styles.buttonContent}
+                                labelStyle={styles.buttonLabel}
+                                onPress={() => { }}
+                                mode='outlined'>
+                                Add Credits
+                            </Button>
+                            <Button
+                                style={styles.button}
+                                contentStyle={styles.buttonContent}
+                                labelStyle={styles.buttonLabel}
+                                onPress={() => { }}
+                                mode='outlined'>
+                                Send Credits
+                            </Button>
+                        </View>}
+                    </View>
+                    <Divider style={styles.separator} />
+                    <DrawerItem
+                        icon={({ color, size }) => (
+                            <Image source={Images.settings} />
+                        )}
+                        label="Account Settings"
+                        labelStyle={styles.optionLabel}
+                        onPress={() => { }} />
+                    <Divider style={styles.separator} />
+                    <DrawerItem
+                        icon={({ color, size }) => (
+                            <Image source={Images.help} />
+                        )}
+                        label="Help & Feedback"
+                        labelStyle={styles.optionLabel}
+                        onPress={() => { }} />
+                    <Divider style={styles.separator} />
+                    <DrawerItem
+                        icon={({ color, size }) => (
+                            <Image source={Images.help} />
```

Comment:
This icon doesn't fit here

Reply:
It's just a temporary placeholder since the logout icon isn't available currently. I'll update it once we have it.